lifemanager.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+#include "boardview.h"
+
+class LifeManager
+{
+public:
+	LifeManager(GLFWwindow* window, BoardView* boardview);
+
+	GLFWwindow* window_;
+	BoardView* boardView_;
+
+	int lives;
```

Comment:
Your variable are not often used elsewhere, you can write getter if you want to access  them and put them in private.